collectors/node.d.plugin/snmp/README.md

```diff
@@ -183,15 +183,17 @@ The `options` given for each server, are:
 
 To use SNMPv3:
 
--   set `version` to 3
 -   use `user` instead of `community`
+-   set `version` to 3
+
 
 User syntax:
 
 ```json
 {
     "user": {
         "name": "userName",
+        "version": 3,
```

Comment:
"version" is a [part of](https://github.com/netdata/netdata/blob/872fc3adb4d43939d8d8268a384807d05e0bc1e2/collectors/node.d.plugin/snmp/README.md?plain=1#L171-L179) "options".

- v2

```json
{
  "enable_autodetect": false,
  "update_every": 10,
  "servers": [
    {
      "hostname": "10.11.12.8",
      "community": "public",
      "update_every": 10,
      "options": {
        "version": 1
      },
      "charts": {
      }
    }
  ]
}
```

- v3

```json
{
  "enable_autodetect": false,
  "update_every": 10,
  "servers": [
    {
      "hostname": "10.11.12.8",
      "user": {
        "name": "userName",
        "level": 3,
        "authProtocol": "3",
        "authKey": "authKey",
        "privProtocol": "2",
        "privKey": "privKey"
      },
      "update_every": 10,
      "options": {
        "version": 3
      },
      "charts": {
      }
    }
  ]
}
```

Reply:
Thanks for bringing this to my attention. :)